stable/registry/templates/deployment.yaml

```diff
@@ -0,0 +1,58 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "fullname" . }}
+  labels:
+    app: {{ template "fullname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: {{.Release.Name | quote }}
+    heritage: {{.Release.Service | quote }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: {{ template "fullname" . }}
```

Comment:
The selector can be dropped IMHO. If it's retained, set `app: {{ template "name" . }}` and add `release: {{ .Release.Name }}`.